src/amo/components/Header/styles.scss

```diff
@@ -41,12 +41,11 @@
     @include text-align-start();
 
     align-self: end;
-    grid-column: 1 / 3;
     grid-row: 1 / 2;
-    margin: 24px 0 0;
+    margin: 40px 0 0;
```

Comment:
Most of the mocks going forward have used `6px` as the base size; should this be `48px` as a multiple of 6?

Reply:
Yeah it could do - that would make a bit more room for the heading to slide under the top-right buttons/links.